vscode-antimony/src/server/bioservices_server/webservices.py

```diff
@@ -92,3 +114,55 @@ def annot_search_uniprot(self, query: str):
                 'prefix': 'uniprot',
             })
         return objects
+    
+    def annot_search_rhea(self, query: str):
+        self.init_rhea()
+        
+        if query.strip() == '':
+            return list()
+
+        try:
+            result_df = self.rhea.search(query, columns='rhea-id,equation')
+        except URLError:
+            raise NetworkError
+
+        result_df = result_df[0:20]

```

Comment:
I'm concerned about the constants in the slice, "[0:20]". Why is it 20? Can this change with release level?

Reply:
20 is just the first 20 relevant info (I think items after 20 should be irrelevant), can be changed if needed.

Reply:
add a named constant